machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/DataSourceConfig.java

```diff
@@ -0,0 +1,49 @@
+package pl.jellytech.machiavelli.cards;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.jellytech.machiavelli.cards.utils.TimeUtils;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class DataSourceConfig extends com.zaxxer.hikari.HikariConfig {
+
+    @Value("${machiavelli.card.datasource.url}")
+    private String url;
+    @Value("${machiavelli.card.datasource.driverClassName}")
+    private String driverClassName;
+    @Value("{machiavelli.card.datasource.username}")
+    private String username;
+    @Value("{machiavelli.card.datasource.password}")
+    private String password;
+    private final int connectionTimeout = 5;
+    private final int IdleTimeout = 2;
```

Comment:
naming - to lower letter

Reply:
and have those connection timeouts in properties - this allows you to always dynamically reconfigure settings on the server without the need of a code change